velox/vector/BaseVector.cpp

```diff
@@ -734,7 +734,10 @@ uint64_t BaseVector::estimateFlatSize() const {
   }
 
   auto leaf = wrappedVector();
-  VELOX_DCHECK_GT(leaf->size(), 0);
+  // This can happen if wrapped vector has nulls in all rows?
+  if (UNLIKELY(leaf->size() == 0)) {
+    return 0;
```

Comment:
I wonder if in case of wrap, we need to add something?
Like nulls if dictionary has nulls...